src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1510,12 +1539,19 @@ internal StyleViewModel(OptionStore optionStore, IServiceProvider serviceProvide
             var predefinedTypesGroupTitle = CSharpVSResources.predefined_type_preferences_colon;
             var varGroupTitle = CSharpVSResources.var_preferences_colon;
             var nullCheckingGroupTitle = CSharpVSResources.null_checking_colon;
+            var usingsGroupTitle = CSharpVSResources.using_preferences_colon;
             var modifierGroupTitle = ServicesVSResources.Modifier_preferences_colon;
             var codeBlockPreferencesGroupTitle = ServicesVSResources.Code_block_preferences_colon;
             var expressionPreferencesGroupTitle = ServicesVSResources.Expression_preferences_colon;
             var variablePreferencesGroupTitle = ServicesVSResources.Variable_preferences_colon;
             var parameterPreferencesGroupTitle = ServicesVSResources.Parameter_preferences_colon;
 
+            var usingDirectivePlacementPreferences = new List<CodeStylePreference>
+            {
+                new CodeStylePreference(CSharpVSResources.Inside_namespace, isChecked: false),
+                new CodeStylePreference(CSharpVSResources.Outside_namespace, isChecked: false),
```

Comment:
i thouht we had 'preserve' as an option.  did that go away?

Reply:
It did. I've been convinced it is better this way. =)

Reply:
ah ok.  i think i like that better :)